xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp

```diff
@@ -174,6 +200,38 @@ void CVideoLayerBridgeDRMPRIME::Configure(CVideoBufferDRMPRIME* buffer)
     m_DRM->AddProperty(plane, "COLOR_RANGE", value);
 
   auto connector = m_DRM->GetConnector();
+
+  int bpc = 8;
+  bool ycc = false;
+  std::string output_format = "Default";
+
+  if (picture.colorBits > 8)
+  {
+    bpc = 12;
+    ycc = true;
+    output_format = "YCrCb422";
+  }
+
+  std::tie(result, value) = connector->GetPropertyValue("Colorspace", GetColorimetry(picture, ycc));
+  if (result)
+  {
+    CLog::Log(LOGDEBUG, "CVideoLayerBridgeDRMPRIME::{} - setting connector colorspace to {}",
+              __FUNCTION__, GetColorimetry(picture, ycc));
+    m_DRM->AddProperty(connector, "Colorspace", value);
+    m_DRM->SetActive(true);
+  }
+
+  CLog::Log(LOGDEBUG, "CVideoLayerBridgeDRMPRIME::{} - setting max bpc to {} ({})", __FUNCTION__,
+            bpc, result);
+  result = m_DRM->AddProperty(connector, "max bpc", bpc);
```

Comment:
```suggestion
  m_DRM->AddProperty(connector, "max bpc", bpc);
```

Reply:
What is the use of setting "max bpc" what is actually happening here that this needs to be set? Why can't the driver auto limit itself if it can configure the clocks to drive the desired output?

Reply:
As stated above, the latest PR does so